### PR TITLE
ENH: Restore carpetplot and other final adjustments

### PR DESCRIPTION
**Pull request by effigies**
## Changes proposed in this pull request

Closing in on the alpha. Getting tests running, but hopefully we'll finish up the reports and cleanup tomorrow.

### CODE REVIEW

File: fmriprep/interfaces/resampling.py
Comment (effigies) [inline fmriprep/interfaces/resampling.py:1]: Completing change started in e14d360baa5a781065c02817ec7a4c52387facb9.

**Comment by effigies** [comment]
This could be reviewed and merged, but I'll keep working on the docs and pushing them until it is.

**Comment by effigies** [comment]
@mgxd I think this is ready to go. It's not a complete overhaul of the documentation, but I think that's okay, given that `--level full` will be the default.

**Review by mgxd — APPROVED** [review]
Took a quick skim and lgtm - when are you targeting the first alpha release? I'm hoping to have a migas-py release by tomorrow...

**Comment by effigies** [comment]
Whenever I can. Need to release smriprep, so can take a day or two to get it out.

**Pull request merged**